code-examples/quarkus-service-registry-quickstart/consumer/.env

```diff
@@ -1,6 +0,0 @@
-## Generated by rhoas generate-config
```

Comment:
I have removed this file. Want to make sure if the `.env` files were added accidentally or do we intend to keep them.